### PR TITLE
Synced firebase with the Accomplished Project Cards and Modal of Home…

### PR DESCRIPTION
**Pull request by vismay12195**
Hi Yagnank,

Just Final touches to the application, adding the Project Cards and upon clicking them created a Project Modal window which will display all the project related details to the user. In the end, I have also made the application fully responsive for all the pages of this Single Page Application. Now as a final step, I will simply deploy this project on Firebase as we discussed and add it to my portfolio as well. Let me know if any last changes to be made.

### CODE REVIEW

**Comment by FRONTGEEK** [comment]
Hey Vismay,
I have checked your code and this last addition of the Project cards and their individual modals look quite intuitive so that any user can go through the accomplished project details. Everything looks perfect and I can also mention that this app is totally scalable, productive and informational. Now just deploy!

File: src/Components/Home/Home.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                <p className={styles.title}>Accomplished Projects</p>
+                <div className={styles.projects}>
+
+                    {
+                        projectsLoaded ? (
+                            projects.length > 0 ? (
+                                projects.map((item) => (
+                                    <div className={styles.project}
+                                        key={item.pid}
+                                        onClick={() => { handleProjectCardClick(item) }}>
```
Comment (FRONTGEEK) [inline src/Components/Home/Home.js:83]: Project card modal functionality is doing what it is supposed to.

**Review by FRONTGEEK — APPROVED** [review]
You did it!

File: src/Components/Home/Home.js (hunk 1 above)
Comment (vismay12195) [reply src/Components/Home/Home.js:83]: Thank you Sir for all the guidance and helping me throughout this project.

**Pull request merged**